projects/addon-table/components/table/thead.directive.ts

```diff
@@ -0,0 +1,31 @@
+import {Directive, HostBinding, HostListener, Inject} from '@angular/core';
+import {
+    INTERSECTION_ROOT_MARGIN,
+    IntersectionObserverService,
+} from '@ng-web-apis/intersection-observer';
+import {asCallable} from '@tinkoff/ng-event-plugins';
+import {Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+@Directive({
+    selector: 'thead[tuiThead]',
+    providers: [
+        IntersectionObserverService,
+        {
+            provide: INTERSECTION_ROOT_MARGIN,
+            useValue: '0px 10000px 10000px 10000px',
+        },
+    ],
+})
+export class TuiTheadDirective {
+    @HostBinding('$.class._stuck')
```

Comment:
Need to go over the whole library and update such bindings. See underline component.